packages/cli/src/zosfiles/compare/lf-uss/LocalfileUss.handler.ts

```diff
@@ -23,6 +23,6 @@ export default class LocalfileUssHandler extends CompareBaseHandler {
         return helper.prepareLocalFile(args.localFilePath);
     }
     public async getFile2(session: AbstractSession, args: ICommandArguments, helper: CompareBaseHelper): Promise<string | Buffer> {
-        return await Get.USSFile(session, args.ussFilePath, { ...helper.file2Options, task: helper.task });
+        return await Get.USSFile(session, args.file, { ...helper.file2Options, task: helper.task });
```

Comment:
Since the command definition for `zowe files compare lf-uss` defines the positional argument as `ussFilePath`, I think we need to keep that name here: https://github.com/zowe/zowe-cli/blob/e7f4484bc132ca0ef6a7cd2249c1330f5dfcafee/packages/cli/src/zosfiles/compare/lf-uss/LocalfileUss.definition.ts#L40-L45

Reply:
![image](https://github.com/zowe/zowe-cli/assets/112635587/70381c31-2ffd-4a23-8320-31a64e9a49be)

Reply:
![image](https://github.com/zowe/zowe-cli/assets/112635587/a796adc1-0258-423b-a2af-aee386e8b425)
![image](https://github.com/zowe/zowe-cli/assets/112635587/b23e4121-6589-4e5f-a5e0-07cb182d471e)
dar
![image](https://github.com/zowe/zowe-cli/assets/112635587/f18cf99e-0f9a-4cc7-b836-6b082f3a0b42)

Reply:
Changing from `ussFilePath` to `file` makes the new command work, but breaks the `files compare lf-uss` command:
```
zowe files compare lf-uss <localFile> <ussFile>
Command Error:
Expect Error: Specify the USS file name.
```